apps/design/backend/src/index.ts

```diff
@@ -61,6 +61,8 @@ async function main(): Promise<number> {
       }
       throw error;
     }
+  } else {
+    await store.syncOrganizationsCache(await auth0.allOrgs());
```

Comment:
Should we just remove the `if (authEnabled())` guard on L50? I've been through so many iterations with this that I can't remember why I added it in the first place lol

Reply:
I know the feeling 😅 
+1 to just removing the condition - will do